components/external_server/tests/routes/test_auth.py

```diff
@@ -92,7 +92,7 @@ def assert_ldap_lookup_connection_created(self, connection_mock):
     def assert_ldap_bind_connection_created(self, connection_mock):
         """Assert that the LDAP bind connection was created with the lookup user dn and password."""
         self.assertEqual(
-            connection_mock.call_args_list[1][1], dict(user=self.USER_DN, password=PASSWORD, auto_bind=True)
+            connection_mock.call_args_list[1][1], dict(user=self.USER_DN, password=PASSWORD, auto_bind='NO_TLS')
```

Comment:
Line 30 is flagged by DeepSource as potentially insecure. Please add "  # skipcq: PY-A6006" to that line to suppress the DeepSource issue.